.claude/agents/documentation.md

```
@@ -29,12 +29,18 @@ mcp__context7__get-library-docs(context7CompatibleLibraryID="<id>")
 - plantuml (architecture diagrams)
 - sphinx (documentation generation)
 
-### Terraform MCP Server - Resource Documentation
+### AWS MCP Server - Resource Documentation
 #### When documenting OpenTofu/Terraform resources:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)